content/learn/book/ecs/_index.md

```diff
@@ -0,0 +1,114 @@
+
+This tutorial aims to give both an awareness and practical examples of the following:
+* There is a startup scheduler and a normal scheduler ({{rust_type(type="struct", crate="bevy_ecs", name="Schedule", no_mod=true, plural=false)}})
+* Startup systems run only ONCE before normal systems
+* You can ask normal systems to run once or in an infinite loop
+* You can slow the infinite loop by specifying an additional `k` second sleep delay between each loop
+  - where `k ∈ [0, 9223372036854775807_i64]` seconds (i.e. ~292 billion years)
+* A **System** results from calling the inline {{rust_type(type="trait" crate="bevy_ecs", name="IntoForEachSystem" method="system" no_struct=false)}} method on a `fn`. It is a {{rust_type(type="struct", crate="bevy_ecs", name="SystemFn", no_mod=true, plural=false)}} struct that implements the {{rust_type(type="trait", crate="bevy_ecs", name="System", no_mod=true, plural=false)}} trait and encapsulates the original function 
+* A **System** is always assigned a **stage** in the scheduler
+  - This dictates the group of systems that it runs in parallel with
+  - The default normal stage is `stage::UPDATE`
+  - The default startup stage is `startup_stage::STARTUP`
+* Except when certain data constraints exist (i.e. thread local with exclusive access to {{rust_type(type="struct", crate="bevy_ecs", name="World", no_mod=true, plural=false)}} and {{rust_type(type="struct", crate="bevy_ecs", name="Resources", no_mod=true, plural=false)}})
+  - Startup systems run in no particular order within their **stage** due to parallel execution
+  - Normal systems run in no particular order within their **stage** due to parallel execution
+* Stages exist as an ordered collection
+* For finer control of the order of execution within a scheduler
+  - You can create as many NEW stages in the scheduler as you like
+  - New stages are added immediately *before* or *after* an existing stage (system defined or custom)
+* You can emit and consume custom events
+* You can add new **Components** and **Systems** to a running {{rust_type(type="struct", crate="bevy_app", name="App", no_mod=true, plural=false)}}, i.e. after `.run()`
+  - with `commands:` {{rust_type(type="struct", crate="bevy_ecs", name="Commands", no_mod=true, plural=false)}}
+* {{rust_type(type="struct", crate="bevy_app", name="App", no_mod=true, plural=true)}} also have a mechanism for exiting gracefully
+
+## Create Project
+
+First [create](@/learn/book/getting-started/setup/_index.md#create-a-new-rust-executable-project) a new project called `bevy_ecs_tut` and [add](@/learn/book/getting-started/setup/_index.md#add-bevy-to-your-project-s-cargo-toml) <img src="/assets/bevy_logo_dark.svg" style="height: 1em;margin-bottom:-0.1em" alt="BEVY"/> as a dependency.
+
+If you don't know or remember the current version of Bevy, `cargo` can fetch it for you fast.
+```bash
+$ cargo search bevy
```

Comment:
We already have a section on setting up a bevy project with the latest version (which we linked to). I don't think we should duplicate content like this. Lets just link to the other section.

Reply:
Do you mind if I drop the `cargo search bevy` tip into `getting-started`?

Reply:
Yup thats fine. Lets add it after the "this is the current Bevy version" section.

"You can also look up the current version using the following cargo command:"